cgroup1/utils.go

```diff
@@ -180,49 +179,18 @@ func parseKV(raw string) (string, uint64, error) {
 // etc.
 //
 // The resulting map does not have an element for cgroup v2 unified hierarchy.
-// Use ParseCgroupFileUnified to get the unified path.
+// Use github.com/containerd/cgroups/v3.ParseCgroupFileUnified to get the unified path.
 func ParseCgroupFile(path string) (map[string]string, error) {
 	x, _, err := ParseCgroupFileUnified(path)
 	return x, err
 }
 
 // ParseCgroupFileUnified returns legacy subsystem paths as the first value,
 // and returns the unified path as the second value.
+//
+// Deprecated: use github.com/containerd/cgroups/v3.ParseCgroupFileUnified .
```

Comment:
Same here

```suggestion
// Deprecated: use [cgroups.ParseCgroupFileUnified} instead.
```

<img width="995" alt="Screenshot 2023-01-30 at 16 00 07" src="https://user-images.githubusercontent.com/1804568/215512939-a0418f64-d573-47e8-953b-a7ffd79f071d.png">